### PR TITLE
Adds asc + desc sorting

### PR DESCRIPTION
**Pull request by Sqyphen**
Adds the ability to reverse the sorting of the images, can be used with published data and colour ordering

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ A list of some interesting expansions for the project:
 
 + ~Sort by (pixel) colour~
```
Comment (Lily2point0) [inline README.md]: Add `sort` to params in "How can I use it?" section

Comment (Sqyphen) [reply README.md]: oops, thanks. This is now fixed.

File: static/js/script.js [hunk 2]
```diff
@@ -7,10 +7,11 @@ var dateTo = document.getElementById('dateTo');
 var timeTo = document.getElementById('timeTo');
 var orientation = document.getElementById('orientation');
 var fit = document.getElementById('fit');
+var order = document.getElementById('order');
 var sort = document.getElementById('sort');
 
 function init(){
-  if(width !== undefined && height !== undefined && dateFrom !== undefined && timeFrom !== undefined && dateTo !== undefined && timeTo !== undefined && orientation !== undefined && fit !== undefined && sort !== undefined){
+  if(width !== undefined && height !== undefined && dateFrom !== undefined && timeFrom !== undefined && dateTo !== undefined && timeTo !== undefined && orientation !== undefined && fit !== undefined && order !== undefined && sort !== undefined){
```
[the remark this answers is not in the document]
Comment (Sqyphen) [reply static/js/script.js]: You're right and that file is getting quite bloated, let me have a look at that...

[the remark this answers is not in the document]
Comment (Sqyphen) [reply static/js/script.js]: Okay, that should be a bit clearer and resilient

File: static/js/script.js [hunk 3]
```diff
@@ -1,44 +1,62 @@
...
+  'timeFrom',
+  'dateTo',
+  'timeTo',
+  'orientation',
+  'fit',
+  'order',
+  'sort'
+];
+
+document.addEventListener("DOMContentLoaded", function() {
```
Comment (Lily2point0) [inline static/js/script.js]: can be shortened to `document.addEventListener("DOMContentLoaded", init);`

Comment (Sqyphen) [reply static/js/script.js]: ah, of course - amended this

**Pull request merged**